lib/cli/ui/prompt.rb

```diff
@@ -132,6 +132,8 @@ def ask_interactive(question, options = nil, multiple: false)
 
           raise(ArgumentError, 'insufficient options') if options.nil? || options.size < 2
           instructions = (multiple ? "Toggle options. " : "") + "Choose with ↑ ↓ ⏎"
+          instructions += ", filter with 'f'" unless (options - %w(yes no)).empty?
```

Comment:
I’d prefer if `filter_ui` and `enter_ui` were options. If `nil`, do this smart behaviour, but allow to force `true` or `false`

Reply:
Would you want these options to totally disable the features or just make it so the instruction text doesn't display?

Reply:
I understand disabling the whole feature would be difficult and I don't see why we should disable the feature, so I guess just changing the message is fine.

You can also remove the `yes/no` special behaviour and just have `confirm` disable the messages.